x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/event_column_view.tsx

```diff
@@ -201,6 +197,18 @@ export const EventColumnView = React.memo<Props>(
         : grouped.icon;
     }, [button, closePopover, id, onClickCb, data, ecsData, timelineActions, isPopoverOpen]);
 
+    const handlePinClicked = useCallback(
```

Comment:
++ Thanks for saving the renders! 🚀 

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/87354116-432c7f80-c51b-11ea-8c92-7184c82e68ac.png" />
</p>